src/main/proto/wfa/measurement/api/v2alpha/exchange_workflow.proto

```diff
@@ -37,15 +37,44 @@ message ExchangeWorkflow {
     // the step-specific name to a label for the input. The step is ready to run
     // when, for each input label, there is a completed step with the same
     // output label.
+    //
+    // Some input labels may have no step that produces that label as an output.
+    // This are the original sources of data. To execute the protocol, the party
+    // will need to provide a configuration showing where these original data
```

Comment:
What is an example of this? What kind of trust are we assuming of the parties for this configuration?

Reply:
Changed this part up a bit, PTAL